doc/source/changes/version_0_26.rst.inc

```diff
@@ -0,0 +1,42 @@
+﻿New features
+------------
+
+* added a feature (see the :ref:`miscellaneous section <misc>` for details).
+
+* added another feature.
+
+.. _misc:
+
+Miscellaneous improvements
+--------------------------
+
+* improved something.
+
+Fixes
+-----
+
+* fixed call to `reindex` by passing an array of labels as `new_axis` argument (closes :issue:`384`):
```

Comment:
I don't consider this to be a bug since the array type wasn't documented as a valid argument type => move to improvements? But well that is subjective :)